qiskit/optimization/problems/quadratic_program.py

```diff
@@ -851,6 +853,216 @@ def substitute_variables(
         """
         return SubstituteVariables().substitute_variables(self, constants, variables)
 
+    def to_ising(self) -> Tuple[OperatorBase, float]:
+        """Return the Ising Hamiltonian of this problem.
+
+        Returns:
+            The qubit operator of the problem and the shift value.
```

Comment:
Could you add an additional sentence explaining what the shift value is? This might not be clear to everyone.